.github/workflows/docs-build-push.yml

```diff
@@ -0,0 +1,33 @@
+name: Build and deploy (docs)
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: 'Environment to deploy to'
+        required: false
+        default: 'preview'
+        type: choice
+        options:
+        - preview
+        - dev
+        - staging
+        - prod
+  pull_request:
+    branches:
+      - "*"
+    paths:
+      - site/**
+
+jobs:
+  call-docs-build-push:
+    uses: nginxinc/docs-actions/.github/workflows/docs-build-push.yml@main
```

Comment:
I think the best practice is to put a commit sha here? Are you happy not to pin to a particular version?

Reply:
Yeah a commit sha here would be better. I left it at `main` since it would let us bump the theme version without creating a PR across all the repos with docs ( https://github.com/nginxinc/docs-actions/blob/03a9a3808fcb77cd0c19d7fa5d59b25565dd1d6d/.github/workflows/docs-build-push.yml#L42). 
I'll update this to a commit sha, and look at storing the theme version externally so we can still update it cleanly.